nextflow_schema.json

```diff
@@ -1,358 +1,320 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "https://raw.githubusercontent.com/epi2me-labs/wf-basecalling/master/nextflow_schema.json",
-    "title": "epi2me-labs/wf-basecalling",
-    "description": "Helper workflow for basecalling ONT reads.",
-    "demo_url": "https://ont-exd-int-s3-euwst1-epi2me-labs.s3.amazonaws.com/wf-basecalling/wf-basecalling-demo.tar.gz",
-    "aws_demo_url": "https://ont-exd-int-s3-euwst1-epi2me-labs.s3.amazonaws.com/wf-basecalling/wf-basecalling-demo/aws.nextflow.config",
-    "url": "https://github.com/epi2me-labs/wf-basecalling",
-    "type": "object",
-    "definitions": {
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Please revert all non-functional whitespace changes so that we can better understand what has actually been changed.